lib/runTx.js

```diff
@@ -69,7 +69,6 @@ module.exports = function (opts, cb) {
 
     if (tx.to.toString('hex') !== '') {
       accounts.add(tx.to.toString('hex'))
-      self.stateManager.touched.push(tx.to)
```

Comment:
Same for this?

Reply:
I'm trying to remember; I think this one is just logically superfluous. Any operation that modifies the empty-ness of the `to` address will necessarily add that address to the touched set. Therefore there's no need to do it here as part of cache population.

Reply:
Ah, just seeing that the ``toAccount`` (also the newly created one from ``loadToAccount``) is added in ``runCall.addTxValue()`` through ``stateManager.putAccount()``. So this should be fine.